addons/sale/data/mail_template_data.xml

```diff
@@ -29,7 +29,28 @@
             </t>
             amounting in <span style="font-weight: bold;" t-out="format_amount(object.amount_total, object.currency_id) or ''">$ 10.00</span> is ready for review.
         </t>
-        <br/><br/>
+        <br/>
+        <t t-set="documents" t-value="object._get_product_documents()"/>
+        <t t-if="documents">
+            <br/>Here 
+            <t t-if="len(documents)>1">
+                are some additional documents
+            </t>
+            <t t-else="">
+                is an additional document
+            </t>
+            that may interest you:
```

Comment:
```suggestion
            <br/> 
            <t t-if="len(documents)>1">
                Here are some additional documents that may interest you:
            </t>
            <t t-else="">
                Here is an additional document that may interest you:
            </t>
```
I know the existing content of the template is a bad example of it, but to ease the work of translators, we should avoid splitting sentences as much as possible.  Depending on the language, the place of words vary a lot and the ordering you did in english may be the opposite in another language.

For mail templates, it's less an issue as the whole template is considered as one 'block' to be translated, but in some other cases, e.g. templates used on website, 'html' content is parsed to extract translatable strings.  This sometimes leads to situations where the translators only get part of sentences, if not one word alone, and it's difficult to understand the meaning of that part, and/or translate it rightly in their own language with the information/restrictions they have.